arm-tool/arm-tool.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>arm_tool</RootNamespace>
```

Comment:
Is deze echt nodig? Want hierdoor zijn al je namespaces een beetje lelijk met `armtool.X`